osu.Game/Beatmaps/BeatmapSetOnlineGenre.cs

```diff
@@ -0,0 +1,20 @@
+﻿// Copyright (c) ppy Pty Ltd <contact@ppy.sh>. Licensed under the MIT Licence.
+// See the LICENCE file in the repository root for full licence text.
+
+namespace osu.Game.Beatmaps
+{
+    public enum BeatmapSetOnlineGenre
+    {
+        Any = 0,
```

Comment:
usually, enum starts at 0, only assign to number when you need, like starting from `HipHop = 9` because it jumped from 7 (hope you understand what i mean)

Reply:
https://github.com/ppy/osu-web/blob/master/database/seeds/ModelSeeders/MiscSeeder.php

Reply:
there's another thing i got confused of, `Any` is 99 on [MiscSeeder.php](https://github.com/ppy/osu-web/blob/master/database/seeds/ModelSeeders/MiscSeeder.php#L33), why are you setting it to 0, also what i mean by my first comment is doing something like this
```csharp
Any = 99, // assign
Unspecified = 1, // assign
VideoGame, // no need to, last one was 1, this should be 2
Anime, // same
...
```
Unless I'm mistaken

Reply:
Because of
```
$any_genre = \App\Models\Genre::find(99);
$any_genre->genre_id = 0;
$any_genre->save();
```

Reply:
Ah, I see

Reply:
> ~https://github.com/ppy/osu-web/blob/master/database/seeds/ModelSeeders/MiscSeeder.php~ nvm, misunderstood what you meant. What's the styleguide for that in osu? I was pretty sure i need to assign every value

I don't see any enum being assigned one by one, unless they are flags (1, 2, 4, 8, 16, ..), or require jump in values due to an enum value that doesn't exist or something like that